libjsonutils/test/src/main.cpp

```diff
@@ -1,6 +1,7 @@
-#include <gtest/gtest.h>
 #include <jsonutils/json_utils.h>
 
+#include <gtest/gtest.h>
```

Comment:
I would stick to the [Google's C++ Style Guide](https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes) and include third party headers before your own

Reply:
Maybe good for Google, but I prefer to check the header to be compilable.
see https://llvm.org/docs/CodingStandards.html#include-style

Reply:
mm ok, but then we need to be consistent everywhere, maybe as part of a second pull request that modifies just the order of includes?

Reply:
ok